app/assets/javascripts/components/students/assign_button.jsx

```diff
@@ -109,6 +109,9 @@ const AssignButton = React.createClass({
 
     if (assignment.title === '' || assignment.title === 'undefined') {
       return;
+    } else if (assignment.title.length > 255) {
+      alert(I18n.t('assignments.title_too_large'));
```

Comment:
We'll also need a corresponding message string in the locale file (`config/locates/en.yml`, with an description of what the message is for in `qqq.yml`).